README.md

```diff
@@ -20,14 +20,38 @@ Your goal is to produce shapes by cutting, rotating, merging and painting parts
 5. If not, file a new issue here: https://github.com/tobspr/shapez.io/issues/new
 6. I will then have a look (This can take days or weeks) and convert it to trello, and comment with the link. You can then vote there ;)
 
-## Building
-
--   Make sure `ffmpeg` is on your path
--   Install Node.js and Yarn
--   Install Java (required for textures)
--   Run `yarn` in the root folder
--   Cd into `gulp` folder
--   Run `yarn` and then `yarn gulp` - it should now open in your browser
+## Building (for windows)
```

Comment:
"Windows" should probably be capitalized

also it *might* be helpful to add that for other systems, setup is whatever equivalent they have, but I don't know much about other systems